bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BridgeHandler.java

```diff
@@ -473,8 +473,7 @@ private void forwardStateToHandlers(DeviceServiceData deviceServiceData, JsonEle
             // All children of this should implement BoschSHCHandler
             @Nullable
             ThingHandler baseHandler = childThing.getHandler();
-            if (baseHandler != null && baseHandler instanceof BoschSHCHandler) {
```

Comment:
I think you will the same line in my new ThingDiscoveryService.
Can you please replace my c&p lines with the nice instance check with cast?